Makefile

```diff
@@ -42,7 +42,13 @@ local: $(BINARIES)
 
 $(BINARIES):
 	mkdir -p $(BIN_DIR)
-	$(BUILDCMD) -o $(BIN_DIR)/$@ $(GOTARGET)/cmd/$@
+
+	mkdir -p $(BIN_DIR)/amd64
```

Comment:
These should probably be in a different target, `cross-build` or something like that